test/testconfig.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  username: 'USERNAME',
-  password: 'PASSWORD',
-  targetBranch: 'BRANCH',
+  username: 'Universal API/uAPI5325526267-a07def41',
```

Comment:
I'm not sure you want to share your credentials with the community

Reply:
Yep this was mistaken.